ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -16,31 +18,87 @@ public abstract class NotificationModelBase : ValidationModel
         }
 
         public Notification Notification { get; set; }
+
         [BindProperty]
         public int? NotificationId { get; set; }
         
         // This can be thrown away once proper banner work completes
         public NotificationStatus NotificationStatus { get; set; }
 
+        [ViewData]
+        public Dictionary<string, List<string>> ErrorDictionary { get; set; }

```

Comment:
The keys in this are the db table names, which means we're making our view logic a bit dependant on db strucut, which feels a bit aweful.. I wonder if in `GetModelStateErrors` we can convert it into a more view-friendly format?